kafka-connect/s2i/resources/openshift-template.yaml

```diff
@@ -36,6 +61,9 @@ objects:
     labels:
       build: ${BUILD_NAME}
     name: ${TARGET_IMAGE_NAME}
+  spec:
+    lookupPolicy:
+      local: true
```

Comment:
What does this do?

Reply:
This should allow you to use the image from Kubernetes resources (such as Deployment or StateFulSet) without specifying the registry URL.